src/common/hooks/useSelectedInvoiceVersion/utils.js

```diff
@@ -0,0 +1,23 @@
+import get from 'lodash/get';
+
+import {
+  CONFIG_ADDRESSES,
+  CONFIG_API,
+  LIMIT_MAX,
+  MODULE_TENANT,
+} from '@folio/stripes-acq-components';
+
+export const getTenantAddresses = (ky) => async () => {
+  const searchParams = {
+    limit: LIMIT_MAX,
+    query: `(module=${MODULE_TENANT} and configName=${CONFIG_ADDRESSES})`,
+  };
+
+  return ky.get(CONFIG_API, { searchParams }).json();
+};
+
+export const getVersionMetadata = (version, entity) => ({
```

Comment:
Looks as a reusable tool for all modules, that will use version history approach. It makes sense to keep it in acq-components lib